README.md

```diff
@@ -79,306 +79,273 @@ int main()
+> GitHub.
+
+#### `BEMAN_EXEMPLAR_BUILD_EXAMPLES`
+
+Enable building examples. Default: ON. Values: { ON, OFF }.
+
 </details>
 
 ## Integrate beman.exemplar into your project
 
-<details>
-<summary> Use beman.exemplar directly from C++ </summary>
-<!-- TODO Darius: rewrite section!-->
-
-If you want to use `beman.exemplar` from your project,
-you can include `beman/exemplar/*.hpp`  files from your C++ source files
+To use `beman.exemplar` in your C++ project,
+you should include relevant headers from `beman.exemplar` in your source files.
 
-```cpp
+```c++
 #include <beman/exemplar/identity.hpp>
 ```
 
-and directly link with `libbeman.exemplar.a`
+You will then link your project to `beman.exemplar`.
 
-```shell
-# Assume /opt/beman.exemplar staging directory.
-$ c++ -o identity_usage examples/identity_usage.cpp \
-    -I /opt/beman.exemplar/include/ \
-    -L/opt/beman.exemplar/lib/ -lbeman.exemplar
+### Produce beman.exemplar static library locally
+
+You can include exemplar's headers locally
+by producing a static `libbeman.exemplar.a` library.
+
+```bash
+cmake --workflow --preset gcc-release
+cmake --install build/gcc-release --prefix /opt/beman.exemplar
 ```
 
-</details>
+This will generate such project structure at `/opt/beman.exemplar`.
 
-<details>
-<summary> Use beman.exemplar directly from CMake </summary>
+```txt
+/opt/beman.exemplar
+├── include
+│   └── beman
+│       └── exemplar
+│           └── identity.hpp
+└── lib
+    └── libbeman.exemplar.a
+```
 
-<!-- TODO Darius: rewrite section! Add examples. -->
+### Linking your project to beman.exemplar with CMake
 
-For CMake based projects, you will need to use the `beman.exemplar` CMake module to define the `beman::exemplar` CMake target:
+For CMake based projects,
+you will need to use the `beman.exemplar` CMake module
+to define the `beman::exemplar` CMake target:
 
 ```cmake
 find_package(beman.exemplar REQUIRED)
 ```
 
-You will also need to add `beman::exemplar`
-to the link libraries of any libraries or executables that include `beman/exemplar/*.hpp` in their source or header file.
+You will also need to add `beman::exemplar` to the link libraries of
+any libraries or executables that include beman.exemplar's header file.
 
 ```cmake
 target_link_libraries(yourlib PUBLIC beman::exemplar)
 ```
 
-</details>
-
-<details>
-<summary> Use beman.exemplar from other build systems </summary>
-
-<!-- TODO Darius: rewrite section! Add examples. -->
+### Linking your project to beman.exemplar with compiler options
 
-Build systems that support `pkg-config` by providing a `beman.exemplar.pc` file.
-Build systems that support interoperation via `pkg-config` should be able to detect `beman.exemplar` for you automatically.
+Compile your project so that it links to the header and library correctly.
 
-</details>
+```bash
+c++ -I/opt/beman.exemplar/include \
+    -L/opt/beman.exemplar/lib -lbeman.exemplar \
+    your_project.cpp -o your_project
+```
```

Comment:
Again, I don't think we should bother documenting how to support direct compiler invocations. I suggest we remove this section.